stac_mjx/main.py

```diff
@@ -72,6 +72,10 @@ def run_stac(
     if stac_cfg.skip_transform == 1:
         logging.info("skipping transform()")
         return fit_path, None
+    elif kp_data.shape[0] % model_cfg["N_FRAMES_PER_CLIP"] != 0:
+        raise ValueError(
+            f"N_FRAMES_PER_CLIP ({model_cfg['N_FRAMES_PER_CLIP']}) must be a factor of the number of mocap frames ({kp_data.shape[0]})"
```

Comment:
I think this would be easier to understand if it says something like Here’s a clearer rewording of the sentence:

"N_FRAMES_PER_CLIP must be chosen such that it divides evenly into the number of mocap frames (n_fit_frames in stac.yaml)."